example/Components/SimpleForm.js

```diff
@@ -26,38 +27,84 @@ const Form = ({
     <div className='formGroup'>
-      <label>Name</label>
-      <input
-        type='text'
-        className={isValid(errors.name) ? '' : 'error'}
+      <label htmlFor='name'>Name</label>
+      <input        
+        id='name'
         name='name'
+        type='text'
+        autoFocus
+        required={required.name}
+        aria-required={required.name ? 'true' : null}
+        aria-errormessage='name-error'
+        aria-invalid={isValid(errors.name) ? null : 'true'}
         value={form.name}
         onChange={updateValueAndValidate}
       />
-      <div className='errorPlaceholder'>{ createErrorMessage(errors.name) }</div>
+      <div 
+        id='name-error'
+        className='errorPlaceholder'
+        aria-live='assertive' 
+        aria-atomic='true'
+        aria-relevant='additions text'>
+          <a href='#name'>{ createErrorMessage(errors.name) }</a>
+      </div>
     </div>
     <div className='formGroup'>
-      <label>Random</label>
+      <label htmlFor='random'>Random</label>
       <input
-        type='text'
-        className={isValid(errors.random) ? '' : 'error'}
+        id='random'
         name='random'
+        type='text'        
+        required={required.random}
+        aria-required={required.random ? 'true' : null}
+        aria-errormessage='random-error'
+        aria-invalid={isValid(errors.random) ? null : 'true'}                
         onBlur={validateValue}
         value={form.random}
         onChange={updateValue}
       />
-      <div className='errorPlaceholder'>{ createErrorMessage(errors.random) }</div>
+      <div 
+        id='random-error' 
+        className='errorPlaceholder'
+        aria-live='assertive' 
+        aria-atomic='true'
+        aria-relevant='additions text'>
+          <a href='#random'>{ createErrorMessage(errors.random) }</a>
+      </div>
+    </div>
+    <div className='formGroup'>
+      <label htmlFor='comment'>Comment</label>
+      <input
+        id='comment'
+        name='comment'
+        type='text'
+        required={required.comment}
+        aria-required={required.comment ? 'true' : null}
+        aria-errormessage='comment-error'
+        aria-invalid={isValid(errors.comment) ? null : 'true'}                
+        onBlur={validateValue}
+        value={form.comment}
+        onChange={updateValue}
+      />
+      <div 
+        id='comment-error' 
+        className='errorPlaceholder'
+        aria-live='assertive' 
+        aria-atomic='true'
+        aria-relevant='additions text'>
+          <a href='#comment'>{ createErrorMessage(errors.comment) }</a>
+      </div>
     </div>
     <button
-      {...{ disabled: disableButtonOption && !valid ? 'disabled' : false }}
-      className={disableButtonOption && !valid ? 'inactive' : 'active' }
+      {...{ 'aria-disabled': disableButtonOption && !valid ? 'true' : null }}      
+      aria-describedby={['name-error', 'random-error', 'comment-error'].join(' ')}
```

Comment:
`aria-disabled` and the `aria-describedby` provides the instructions to screen readers that the button will not submit and why. Styling should be based on the `aria-disabled` attribute.